roles/debian/varnish_config/defaults/main.yml

```diff
@@ -1,6 +1,12 @@
 ---
 # Defaults file for varnish_config, other variables exist from importing geerlingguy.varnish and can be overriden
 
+varnish_pidfile: /run/varnishd.pid
+limit_nofile: 131072
```

Comment:
this already exists as varnish_limit_nofile upstream https://github.com/geerlingguy/ansible-role-varnish/blob/master/defaults/main.yml#L16

Please go through all variables and use the upstream name to avoid confusion.

Also: this is the same value as the upstream value. So we don't need to define it again here?